chain/index/ddls.go

```diff
@@ -52,37 +52,40 @@ var ddls = []string{
-		&ps.updateTipsetToRevertedStmt:            "UPDATE tipset_message SET reverted = 1 WHERE tipset_key_cid = ?",
-		&ps.removeTipsetsBeforeHeightStmt:         "DELETE FROM tipset_message WHERE height < ?",
-		&ps.removeEthHashesOlderThanStmt:          "DELETE FROM eth_tx_hash WHERE inserted_at < datetime('now', ?)",
-		&ps.updateTipsetsToRevertedFromHeightStmt: "UPDATE tipset_message SET reverted = 1 WHERE height >= ?",
-		&ps.updateEventsToRevertedFromHeightStmt:  "UPDATE event SET reverted = 1 WHERE message_id IN (SELECT message_id FROM tipset_message WHERE height >= ?)",
-		&ps.isIndexEmptyStmt:                      "SELECT NOT EXISTS(SELECT 1 FROM tipset_message LIMIT 1)",
-		&ps.getMinNonRevertedHeightStmt:           "SELECT MIN(height) FROM tipset_message WHERE reverted = 0",
-		&ps.hasNonRevertedTipsetStmt:              "SELECT EXISTS(SELECT 1 FROM tipset_message WHERE tipset_key_cid = ? AND reverted = 0)",
-		&ps.updateEventsToRevertedStmt:            "UPDATE event SET reverted = 1 WHERE message_id IN (SELECT message_id FROM tipset_message WHERE tipset_key_cid = ?)",
-		&ps.updateEventsToNonRevertedStmt:         "UPDATE event SET reverted = 0 WHERE message_id IN (SELECT message_id FROM tipset_message WHERE tipset_key_cid = ?)",
-		&ps.getMsgIdForMsgCidAndTipsetStmt:        "SELECT message_id FROM tipset_message WHERE tipset_key_cid = ? AND message_cid = ? AND reverted = 0",
-		&ps.insertEventStmt:                       "INSERT INTO event (message_id, event_index, emitter_addr, reverted) VALUES (?, ?, ?, ?) ON CONFLICT (message_id, event_index) DO UPDATE SET reverted = 0",
-		&ps.insertEventEntryStmt:                  "INSERT INTO event_entry (event_id, indexed, flags, key, codec, value) VALUES (?, ?, ?, ?, ?, ?)",
-		&ps.hasNullRoundAtHeightStmt:              "SELECT NOT EXISTS(SELECT 1 FROM tipset_message WHERE height = ?)",
-		&ps.getNonRevertedTipsetAtHeightStmt:      "SELECT tipset_key_cid FROM tipset_message WHERE height = ? AND reverted = 0 LIMIT 1",
-		&ps.countTipsetsAtHeightStmt:              "SELECT COUNT(CASE WHEN reverted = 1 THEN 1 END) AS reverted_count, COUNT(CASE WHEN reverted = 0 THEN 1 END) AS non_reverted_count FROM (SELECT tipset_key_cid, MAX(reverted) AS reverted FROM tipset_message WHERE height = ? GROUP BY tipset_key_cid) AS unique_tipsets",
-		&ps.getNonRevertedTipsetMessageCountStmt:  "SELECT COUNT(*) FROM tipset_message WHERE tipset_key_cid = ? AND reverted = 0 AND message_cid IS NOT NULL",
-		&ps.getNonRevertedTipsetEventCountStmt:    "SELECT COUNT(*) FROM event WHERE reverted = 0 AND message_id IN (SELECT message_id FROM tipset_message WHERE tipset_key_cid = ? AND reverted = 0)",
-		&ps.hasRevertedEventsInTipsetStmt:         "SELECT EXISTS(SELECT 1 FROM event WHERE reverted = 1 AND message_id IN (SELECT message_id FROM tipset_message WHERE tipset_key_cid = ?))",
+		&ps.getNonRevertedMsgInfoStmt:                 "SELECT tipset_key_cid, height FROM tipset_message WHERE message_cid = ? AND reverted = 0 LIMIT 1",
```

Comment:
@akaladarshi Why is the diff so big here ? Makes it hard to review the change. Please can you fix this ?

Reply:
The only change we should see is the new `getNonRevertedTipsetEventEntriesCountStmt` query.

Reply:
@akaladarshi 

Please can you put the `getNonRevertedTipsetEventEntriesCountStmt` statement at the bottom rather than inserting it before `asRevertedEventsInTipsetStmt` and re-fmt ? I think that will reduce the diff.

Reply:
I don't think that will work as well, formatting happens based on the largest variable size. Otherwise it will look odd.

Anyways, I have move towards the end but result is the same.